docs/modules/ROOT/pages/index.adoc

```diff
@@ -8,4 +8,4 @@ The conversion can happen in 2 flavors:
 
 . As a xref:plugin:introduction.adoc[Maven plugin]: AsciiDoc files are converted at full Asciidoctor power independently from Maven site,
 
-. As a xref:plugin:introduction.adoc[Maven site module]: AsciiDoc files are integrated with https://maven.apache.org/doxia/[Maven Doxia] tools, however, with a few limitations.
+. As a xref:site-integration:site-integration.adoc[Maven site module]: AsciiDoc files are integrated with https://maven.apache.org/doxia/[Maven Doxia] tools, however, with a few limitations.
```

Comment:
I think a module should always have an index.adoc page, so you might also considering renaming the site-integration.adoc file to index.adoc.